src/vswhere.lib/LegacyInstance.cpp

```diff
@@ -0,0 +1,130 @@
+// <copyright file="LegacyInstance.cpp" company="Microsoft Corporation">
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt in the project root for license information.
+// </copyright>
+
+#include "stdafx.h"
+
+using namespace std;
+
+template <class T>
+inline HRESULT NotImplemented(_In_ T* p)
```

Comment:
Maybe name this NotFound?  My initial reaction when reading this was that it would return E_NOTIMPL